packages/common/src/PropertyExt.ts

```diff
@@ -296,6 +296,15 @@ export class PropertyExt extends Class {
         }
     }
 
+    resolvePublishedProxy(meta) {
+        let item = this;
+        while (meta.type === "proxy") {
+            item = item[(meta as any).proxy];
```

Comment:
Isn't meta already `any`?

Reply:
No it has a real type (Meta) - that line should really be:
```
item = item[(meta as MetaProxy).proxy]
```
IOW its really:
```
        while (meta instanceof MetaProxy) {
            item = item[meta.proxy];
```

Reply:
Changes made